packages/blocks/src/frame-block/frame-model.ts

```diff
@@ -7,6 +7,7 @@ export const FrameBlockSchema = defineBlockSchema({
   props: () => ({
     xywh: '[0,0,720,480]',
     background: FRAME_BACKGROUND_COLORS[0],
+    zIndex: 0,
```

Comment:
We need to use `string` type for this, so that we can use the `generateKeyBetween` util to handle [fractional indexing](https://github.com/rocicorp/fractional-indexing). This can help making less index updates on switching layer sequence, see [this figma blog post](https://www.figma.com/blog/realtime-editing-of-ordered-sequences/#fractional-indexing)